fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/load/ChunkWriter.java

```diff
@@ -219,7 +220,7 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
 
                                 FHIRPersistenceEvent event = new FHIRPersistenceEvent(fhirResource, props);
 
-                                // Set up the persistence context to include deleted resources when we read
+                                // Set up the persistence context to include NOT include the deleted resources when we read
```

Comment:
```suggestion
                                // Set up the persistence context to NOT include the deleted resources when we read
```

Reply:
the comment says we're gonna stop including deleted resources, but line 224 still retrieves the deleted instances.

I think we actually need to retrieve the deleted instance so that we can set the appropriate versionId.

Reply:
Yes, we do. Thanks for catching this